metadata-drivers/etcd/src/main/java/org/apache/bookkeeper/metadata/etcd/EtcdMetadataClientDriver.java

```diff
@@ -72,4 +72,12 @@ public synchronized void close() {
         }
         super.close();
     }
+
+    @Override
+    public void setSessionStateListener(SessionStateListener sessionStateListener) {
+        /*
+         * TODO: EtcdMetadataClientDriver has to implement this method.
+         */
+        throw new UnsupportedOperationException();
```

Comment:
@codingwangqi im just throwing exception for now. As part of metadataclientdriver/etcd changes can you please take care of this.

@sijie

Reply:
@reddycharan thank you for letting me know about this. I will take care of it.